src/app/Console/Commands/Addons/RequirePro.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Backpack\CRUD\app\Console\Commands\Addons;
+
+use Illuminate\Console\Command;
+
+class RequirePro extends Command
+{
+    use \Backpack\CRUD\app\Console\Commands\Traits\PrettyCommandOutput;
+    use \Backpack\CRUD\app\Console\Commands\Traits\AddonsHelper;
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'backpack:require:pro
+                                {--debug} : Show process output or not. Useful for debugging.';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Require Backpack Pro';
+
+    /**
+     * Backpack addons install attribute.
+     *
+     * @var array
+     */
+    public static $addon = [
+        'name' => 'Backpack Pro',
+        'description' => [
+            'Adds 5 operations, 10 filters, 28 fields, 6 columns, charts',
+        ],
+        'path' => 'vendor/backpack/pro',
+        'command' => 'backpack:require:pro',
+    ];
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed Command-line output
+     */
+    public function handle()
+    {
+        $this->newLine();
+        $this->infoBlock($this->description);
```

Comment:
Hmmm... I don't know if this info block is a good idea here:
- 20% use case - if you run this command individually - maybe it makes sense; though... you know what's going to happen, you typed the command, so even then it might be superfluous;
- 80% use case - if you run this command as part of `php artisan backpack:install`, the info block makes this command look like it's at the same level as `Install Backpack add-ons?` which is a bit confusing:

![CleanShot 2022-08-01 at 08 09 52](https://user-images.githubusercontent.com/1032474/182076626-a5c9ac75-1c64-45e7-a58e-3975cf7552c9.png)

The way I see it as a user, installing one of the premium add-ons is part of the question "_Would you like to install a Backpack add-on?_" so nothing I do there should be at the same level as that question, but _part of it_. Right?

Reply:
Additionally, I love the line that says "running" while installing it, but after it's done, the "Running" line transforms to "DONE", which is again superfluous because there will be a list below with all premium add-ons, where the current add-on will be marked as INSTALLED.

Reply:
See my next comment and image, please, that will be even more explicit.